weatherapplication.client/src/app/routing.module.ts

```diff
@@ -1,23 +1,19 @@
-import { NgModule } from "@angular/core";
-import { RouterModule, Routes } from "@angular/router";
-import { RandomWeatherComponent } from "./core/pages/random-weather/random-weather.component";
-import { CurrentWeatherPageComponent } from "./core/pages/current-weather-page/current-weather-page.component";
-import { FivedaysWeatherPageComponent } from "./core/pages/fivedays-weather-page/fivedays-weather-page.component";
-import { StatisticsPageComponent } from "./core/pages/statistics-page/statistics-page.component";
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { RandomWeatherComponent } from './core/pages/random-weather/random-weather.component';
+import { CurrentWeatherPageComponent } from './core/pages/current-weather-page/current-weather-page.component';
+import { FivedaysWeatherPageComponent } from './core/pages/fivedays-weather-page/fivedays-weather-page.component';
```

Comment:
And a typo again.

Reply:
![image](https://github.com/Oleksandr-Ryabishchuk/WeatherApplication/assets/36698610/41f4e94c-49e7-4678-adbd-9bea90a8af1c)